addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -0,0 +1,177 @@
+from odoo import models, fields, _
+from odoo.exceptions import UserError
+from .account_edi_proxy_auth import OdooEdiProxyAuth
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.fernet import Fernet
+import requests
+import uuid
+import base64
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+
+SERVER_URL = 'https://iap-services.odoo.com'
```

Comment:
I believe there's a new naming scheme to be followed here: `https://l10n-it-edi.api.odoo.com`